### PR TITLE
Retry on Gateway timeout error

### PR DESCRIPTION
**Pull request by AdamVyborny**
JIRA: https://keboola.atlassian.net/browse/COM-1713

### CODE REVIEW

File: src/Api/Api.php [hunk 1]
```diff
@@ -349,7 +353,10 @@ private function executeWithRetry(string $method, string $uri, array $params = [
             return false;
         }, self::RETRY_MAX_ATTEMPTS);
         $proxy = new RetryProxy($retryPolicy, $backOffPolicy);
-        return $proxy->call(function () use ($method, $uri, $params, $body) {
+        return $proxy->call(function () use ($method, $uri, $params, $body, $proxy) {
+            if ($proxy->getTryCount() > 1) {
```
Comment (ondrajodas) [inline src/Api/Api.php]: takhle neee... do toho RetryProxy pošli logger a ono to bude logovat automaticky
https://github.com/keboola/retry/blob/master/src/RetryProxy.php#L83

**Review by ondrajodas — CHANGES_REQUESTED** [review]
do RetryProxy se dá poslat Logger

File: src/Api/Api.php (hunk 1 above)
Comment (AdamVyborny) [reply src/Api/Api.php]: Aha, super. Díky!

**Review by ondrajodas — APPROVED** [review]
takhle super 👍

**Pull request merged**